src/playermat/Playermat.ttslua

```diff
@@ -345,8 +345,17 @@ function doUpkeep(_, clickedByColor, isRightClick)
       discardListOfObjects({ obj })
     elseif obj.hasTag("UniversalToken") == true and obj.is_face_down then
       obj.flip()
-    elseif obj.type == "Card" and not inArea(self.positionToLocal(obj.getPosition()), INVESTIGATOR_AREA) then
+    elseif obj.type == "Card" then
       local cardMetadata = JSON.decode(obj.getGMNotes()) or {}
+      
```

Comment:
For slightly better performance, the metadata decoding should be moved below the "area check"